docker/php/docker-entrypoint.sh

```diff
@@ -7,16 +7,6 @@ if [ "${1#-}" != "$1" ]; then
 fi
 
 if [ "$1" = 'php-fpm' ] || [ "$1" = 'bin/console' ]; then
-	mkdir -p var/cache var/log public/media
-	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var public/media
-	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var public/media
-
-	if [ "$APP_ENV" != 'prod' ]; then
-		composer install --prefer-dist --no-progress --no-suggest --no-interaction
-		bin/console assets:install --no-interaction
-		bin/console sylius:theme:assets:install public --no-interaction
-	fi
```

Comment:
all these commands already ran during the container build. so, if you need to update some file, just run `docker-composer build php` to update files inside the container